### DISCUSSION TITLE
Separate Player List

### DISCUSSION THREAD

**Discussion by BlcDragon** [Minestom/Minestom#2417] [Help]
Hi, is there a ready way to split the player list and make it separate for each instance.
I want each instance to have only the players who are currently in it in the player list.
I know about PlayerInfo packets, but it seems a bit tedious
It would be great if there was an easier way to do this.

**Comment by breitwan** [comment]
You may be interested in #2143

**Comment by DasBabyPixel** [comment]
Oldest open issue in the repo actually #120
